about.md

```diff
@@ -0,0 +1,44 @@
+---
+breadcrumb: About Us
+---
+
+# About Us
+
+Welcome to the 'About Us' page. We are a dedicated participant in the global cybersecurity landscape, contributing our efforts towards the development of reliable encryption tools. Our dedicated team is at the helm of the OpenSSL project, ensuring secure digital interactions. Here, you'll meet the individuals who are collectively shaping a safer, privacy-respecting digital future.
```

Comment:
This text talks about "reliable encryption tools" which is rather specific. Our mission talks more generically about "security and privacy tools".

Reply:
Agree